### ISSUE TITLE
3.1.1 background is now white

### ISSUE THREAD

**Issue by jrausch12** [widget-/slack-black-theme#50]
This may be a possible duplicate issue. The background for 3.1.1 is now white using the instructions on the readme, adding in the custom.css file, and adding the small bit of code from https://github.com/widget-/slack-black-theme/issues/21#issuecomment-342551435 

System is Windows 7 professional

**Comment by mchodson** [comment]
same here. osx 10.12.6. sigh. slack sucks.

**Comment by caiceA** [comment]
same with me

**Comment by caiceA** [comment]
on mac

**Comment by samb0t** [comment]
Yeah, this seems to be a new issue as of the latest slack update.

**Comment by david1602** [comment]
Can confirm, but #48 fixed it for me.

**Comment by samb0t** [comment]
One potential workaround is to add this to the customCustomCSS variable in `index.js`:

`    .c-virtual_list__scroll_container {
        background-color: <your color>;
    }
`

**Comment by Morydees** [comment]
Yep, issue has returned in 3.1.1 despite working for a bit. What a pain in the ass this software has become, can't even get a simple background change...

**Comment by d-fay** [comment]
Trying to keep this fork up-to-date as much as possible with the OneDark theme. Cleaned up a bunch of nuances and added a bash script that can be ran whenever the dark theme breaks after a slack update.

https://github.com/d-fay/slack-black-theme#installing-into-slack

**Comment by Morydees** [comment]
d-fay's fix above worked great on 3.1.1 and offers some additional color options by changing the file text, thanks so much!

**Comment by ngesnetworks** [comment]
Confirmed that @d-fay's fixed it!

**Comment by tiagocedrim** [comment]
@d-fay's fixed it! You rock m8!

**Comment by jrausch12** [comment]
@d-fay I have implemented that fix, however my sidebars are now gray. Please advise.

**Comment by taranu** [comment]
@d-fay your fork points to the master branch's custom.css instead of your fork's copy; not sure if you intended that. But more to the point, can you change the cssPath from an http(s) URL to a local file? I tried to do so and it changed the main window but none of the other top/side panels.
